Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -382,14 +368,13 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "tussen 16:00 en 17:00 vandaag",
```

Comment:
Can you please confirm this is the same behaviour as in English at the TimePeriodParser level?
Also, please include the original case in the DateTimeModel.json, as there potentially the two entities should be merged (and if they aren't, the spec will document the expected behaviour).

Reply:
confirm, as shown in the comment above, Dutch and English behave in the same way given the same type of input.

Reply:
or if you refer to "vandaag" not being included in the extracted text, it happens the same in English with the input 
"I'll be out between 4pm and 5pm today" which extracts "between 4pm and 5pm".